firebase-perf/src/main/java/com/google/firebase/perf/util/Timer.java

```diff
@@ -43,23 +43,17 @@ public class Timer implements Parcelable {
    * compute duration between 2 timestamps in the same timebase. It is NOT wall-clock time.
    */
   private long elapsedRealtimeMicros;
-  /**
-   * Monotonic time measured in the {@link SystemClock#uptimeMillis()} timebase. Only used to
-   * compute duration between 2 timestamps in the same timebase. It is NOT wall-clock time.
-   */
-  private long uptimeMicros;
 
   /**
    * Returns a new Timer object as if it was stamped at the given elapsedRealtime. Uses current
    * wall-clock as a reference to extrapolate the wall-clock at the given elapsedRealtime.
    *
    * @param elapsedRealtimeMillis timestamp in the {@link SystemClock#elapsedRealtime()} timebase
    */
-  public static Timer ofElapsedRealtime(final long elapsedRealtimeMillis, final long uptimeMillis) {
-    long uptimeMicros = MILLISECONDS.toMicros(uptimeMillis);
+  public static Timer ofElapsedRealtime(final long elapsedRealtimeMillis) {
```

Comment:
NIT (not for now) I got confused with the class name "Timer" here (with [this](https://developer.android.com/reference/java/util/Timer)).

Should we rename this as "TraceTime"/"TraceTimestamp" just to avoid ambiguity?

Reply:
Yea personally I also did not like the name `Timer`. This class is more like a Timestamp than a Timer, but I just kept it the same.